docs/en/get_started/installation.md

```diff
@@ -35,10 +35,17 @@ pip install -U openmim
 mim install mmengine
 ```
 
+If you only want to use fileio, registry and configs in MMEngine, you can install mmengine lite:
```

Comment:
```suggestion
If you only want to use the fileio, registry, and config modules in MMEngine, you can set the `MMENGINE_LITE=1` environment variable, which will only install the few third-party library dependencies that are necessary (e.g., it will not install opencv, matplotlib):
```